dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -1110,9 +1111,9 @@ private void importSubProcess(User loginUser, Project targetProject, ArrayNode j
 
             String subProcessJson = JSONUtils.toJsonString(processService.genProcessData(subProcess));
             //check current project has sub process
-            ProcessDefinition currentProjectSubProcess = processDefinitionMapper.queryByDefineName(targetProject.getCode(), subProcess.getName());
+            Boolean ifExist = processDefinitionMapper.existDefByProjectCodeAndDefineName(targetProject.getCode(), subProcess.getName());
```

Comment:
It is best to follow the rules of variable naming, in addition, it is better to use boolean

Reply:
thanks. fixed.